app/services/teacher_training_public_api/sync_sites.rb

```diff
@@ -34,6 +34,7 @@ def perform(provider_id, recruitment_cycle_year, course_id)
         end
 
         handle_course_options_with_invalid_sites(sites)
+        handle_course_options_with_reinstated_sites(sites)
```

Comment:
Do these lines necessarily need to be inside the sites loop?

Reply:
No. Good spot!